libr/debug/p/debug_bf.c

```diff
@@ -168,6 +168,7 @@ static RList *r_debug_native_map_get(RDebug *dbg) {
 	RIOBdescbg *o = dbg->iob.io->desc->data;
 	BfvmCPU *c = o->bfvm;
 	RList *list = r_list_new ();
```

Comment:
Saw a few of these. I'll might do a sweep at a later date to fix all these in that case